scripts/linking.py

```diff
@@ -569,14 +586,18 @@ def eval_candidate_generation_and_linking(examples: List[data_util.MedMentionExa
             doc_mention_no_links_count = 0  # number of ner mentions that did not have any linking candidates
             doc_num_candidates = []
             doc_num_filtered_candidates = []
+            doc_linking_correct_count = 0
+            doc_linking_golds_in_candidates = 0
+            doc_linking_total_predictions = 0
 
             all_golds_per_doc_set = []
             all_golds = []
             all_mentions = []
 
-            classifier_correct_predictions = defaultdict(int)
-            classifier_wrong_predictions = defaultdict(int)
-
+            gold_entities_linker_correct = defaultdict(int)
+            gold_entities_linker_incorrect = defaultdict(int)
+            predicted_entities_linker_correct = defaultdict(set)
```

Comment:
This is a little confusing because it's different from whether we are using `gold_mentions` or not, is there a better name? Maybe not...

Reply:
agreed, however i can't come up with anything better...

Reply:
i'll just add a comment explaining